tor/forward.go

```diff
@@ -0,0 +1,299 @@
+package tor
```

Comment:
Can an integration test be added like https://github.com/cretz/bine/blob/master/tests/tor_listen_test.go? Just something to make sure it works.

Reply:
Done